ci-operator/step-registry/rh-ecosystem-edge/assisted-chat/clean-up/rh-ecosystem-edge-assisted-chat-clean-up-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+OC_URL="https://api.stage.openshift.com/api/assisted-install/v2"
+
+n=4 # number of hours to subtract
+current_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+past_time=$(date -u -d "$current_time -$n hours" +"%Y-%m-%dT%H:%M")
+
+echo "Current UTC time: $current_time"
+echo "Time $n hours ago (UTC): $past_time"
+
+if ! command -v ocm >/dev/null 2>&1; then \
+  mkdir -p "${HOME}/.local/bin" && \
+  curl -sSL -o "${HOME}/.local/bin/ocm" "https://github.com/openshift-online/ocm-cli/releases/latest/download/ocm-linux-amd64" && \
```

Comment:
We use the API directly (curl) with the secret and id elsewhere, right? Let's just do the same here and not have to download this binary.